pkg/container/types/types.go

```diff
@@ -79,8 +79,9 @@ const (
 )
 
 const (
-	TxnTsSize = 12
-	RowidSize = 16
+	TxnTsSize  = 14
+	TxnTidSize = 32
```

Comment:
I think you need at least update container/types/txnts.go -- that is where the TS is implemented.   Unfortunately it has some magic contants, that won't work with simply update TxnTsSize.

Reply:
Do you want to make Tid a type?   Like Rowid is [RowidSize]byte, make this a [TxnTidSize]byte, and update correct places.

Reply:
will  remove the Tid type,  use TxnMeta.ID which is a byte slice .